ui/src/components/NavDrawer.js

```diff
@@ -84,6 +86,9 @@ function ListItemLink(props) {
 
 function NavDrawer() {
   const classes = useStyles();
+  const {
+    state: { useRloc },
```

Comment:
```suggestion
    state: { useGetRUN },
```

Better to change this now than to take on tech debt.

Reply:
Updated here and everywhere. I changed the url param to just "gr=true" for convenience though.